_layouts/product-get-started.html.slim

```diff
@@ -73,18 +73,18 @@ h2#get-started Get Started
     h3.divider#ExtraSection #{page._extra_section_title}
   = page._extra_section
 
-- if page.product.featured_items
+div(ng-app="dcp")
+  .developer-materials(ng-controller="developerMaterialsController")
+    - if page.product.featured_items
+      h3.divider#try-quickstart Try a quickstart or example. Watch a video
+      = partial('developer-materials-featured.html.slim', {'featured_items' => page.product.featured_items})
+      .developer-materials-results data-developer-materials-filter-project-code="#{page.product.dcp_project_code}" ng-init="filters.project = '#{page.product.dcp_project_code}'"
+        = partial 'developer-materials-results.html.slim'
 
-  h3.divider#try-quickstart Try a quickstart or example. Watch a video
+      p: a(href="../developer-materials") More Quickstarts and Developer Materials for #{page.product.abbreviated_name} >>
 
-  = partial('developer-materials-featured.html.slim', {'featured_items' => page.product.featured_items})
-  .developer-materials-results data-developer-materials-max-results="3" data-developer-materials-filter-project-code="#{page.product.dcp_project_code}"
```

Comment:
Notice in the old code, there was a limit of 3. This needs carrying forward, as at present all the results are shown.